recipes/virtual_dataframe/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "virtual_dataframe" %}
+{% set version = "v0.1rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # See https://docs.github.com/en/rest/repos/contents#download-a-repository-archive-tar
+  url: https://github.com/pprados/virtual_dataframe/tarball/{{ version }}/virtual_dataframe.tar.gz
+  sha256: d97bfb35373af61c5fa4269cc9c59d820923823e2134e69c6afe60c3d4593383
+
+build:
+  # noarch: python  # Impossible with entry_points. See https://github.com/conda/conda-build/issues/2627
+  number: 0
+  # no such file directory sur PYTHON ?
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: python -m pip install --progress-bar off --quiet --no-input . -vv
+
+requirements:
+  build:
+    - pip
```

Comment:
This is wrong. See our example recipe https://github.com/conda-forge/staged-recipes/blob/e52ef7ec36fb88f1c00eda0a8c2600815851fe34/recipes/example/meta.yaml#L46-L48